### ISSUE TITLE
Remove workaround added in https://github.com/dotnet/roslyn/pull/26221 once compiler editorconfig work is done

### ISSUE THREAD

**Issue by heejaechang** [dotnet/roslyn#26250]
we should revert work done in https://github.com/dotnet/roslyn/pull/26221 once on-going compiler editorconfig work is done.

**Comment by CyrusNajmabadi** [comment]
@heejaechang Is there still work to be done here.  Afaik all compiler work around editorconfig was done.  Can you clarify what we need to do now?

**Comment by heejaechang** [comment]
if editorconfig file is now part of snapshot and changing that file causes us to have a new snapshot, then the original code is no longer needed.

**Comment by CyrusNajmabadi** [comment]
Can you let me know what specific code we can remove now? Thanks! :-)

**Comment by CyrusNajmabadi** [comment]
I don't have any idea what this is in reference to.  @jasonmalinowski do you know if there's cleanup work we need to do here?

**Comment by CyrusNajmabadi** [comment]
Closing out due to lack of response.
